docs/core/index.md

```diff
@@ -68,3 +68,9 @@ Log out of the currently authenticated partner organization and store. The `logo
 $ shopify logout
 ```
 
+## `config analytics`
```

Comment:
```suggestion
## `config`
```

Two things!
1. To match our existing patterns, the heading should be for the command, not jump directly to the `analytics` subcommand — let's do that in a nested H3 section (see below)
2. With the exception of `help` (due to its foundational importance) the rest of the commands are alphabetical, so this should be moved up, above `connect`.